src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -386,4 +391,4 @@ Um [elemento `<slot />`](/pt-br/basics/astro-components/#slots) dentro de um com
 
 import ReadMore from '~/components/ReadMore.astro';
 
-<ReadMore>Aprenda sobre como utilizar [componentes de frameworks JavaScript](/pt-br/guides/framework-components/) em seu projeto Astro.</ReadMore>
+<ReadMore>Leia mais sobre como utilizar [componentes de frameworks de UI](/pt-br/guides/framework-components/) em seu projeto Astro.</ReadMore>
```

Comment:
We use "leia mais" more often than we use "aprenda"; original also says "UI framework components", no mentions to Javascript in this part.